docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -128,14 +128,18 @@ type Asset = {
 
 ### Multipart Manifest Response
 
-A manifest response of this format is defined by the `multipart/mixed` MIME type as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046#section-5.1). Each part is defined as follows:
+A manifest response of this format is defined by the `multipart/mixed` MIME type as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046#section-5.1).
+
+Headers for this response format are the same as [manifest response headers](#manifest-response-headers), with the exception of `content-type`, which is specified in the `multipart/mixed` MIME type as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046#section-5.1).
```

Comment:
"Media type" is the standard name for these types (since MIME referred to email):
```suggestion
Headers for this response format are the same as [manifest response headers](#manifest-response-headers), with the exception of `content-type`, which is specified in the `multipart/mixed` media type as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046#section-5.1).
```
(Also is "specified in" what you meant to say here, or something else?)

Reply:
```suggestion
Headers for this response format are the same as [manifest response headers](#manifest-response-headers), with the exception of `content-type`, which is specified as the `multipart/mixed` media type as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046#section-5.1).
```